subsys/bluetooth/host/classic/avdtp.c

```diff
@@ -58,16 +57,26 @@ K_MUTEX_DEFINE(avdtp_mutex);
 #define AVDTP_UNLOCK() k_mutex_unlock(&avdtp_mutex)
 
 enum sep_state {
-	AVDTP_IDLE = 0,
-	AVDTP_CONFIGURED,
+	AVDTP_IDLE = 0x01U,
+	AVDTP_CONFIGURED = 0x02U,
 	/* establishing the transport sessions. */
-	AVDTP_OPENING,
-	AVDTP_OPEN,
-	AVDTP_STREAMING,
-	AVDTP_CLOSING,
-	AVDTP_ABORTING,
+	AVDTP_OPENING = 0x04U,
+	AVDTP_OPEN = 0x08U,
+	AVDTP_STREAMING = 0x10U,
+	AVDTP_CLOSING = 0x20U,
+	AVDTP_ABORTING = 0x40U,
 };
 
+static void bt_avdtp_set_state(struct bt_avdtp_sep *sep, uint8_t state)
+{
+	sep->state = state;
+	if (state != AVDTP_IDLE) {
+		sep->sep_info.inuse = 1u;
+	} else {
+		sep->sep_info.inuse = 0u;
+	}
```

Comment:
Why are you using capital `U` in some places but lower-case `u` in others?

Reply:
Change all to `U`